proto/vtctldata.proto

```diff
@@ -857,3 +857,34 @@ message ValidateShardRequest {
 message ValidateShardResponse {
   repeated string results = 1;
 }
+message ValidateSchemaKeyspaceRequest {
```

Comment:
```
❯ echo "ValidateShardResponse\nValidateSchemaKeyspace" | sort
ValidateSchemaKeyspace
ValidateShardResponse
```

:)

Reply:
(same for the rest, to be explicit)

Reply:
```
SFO-M-MAKINJE01:chef-repo makinje$ printf "ValidateSchemaKeyspaceRequest\nValidateSchemaKeyspaceResponse\nValidateSchemaShardRequest\nValidateSchemaShardResponse\nValidateVSchemaRequest\nValidateVSchemaResponse\nValidateRequest\nValidateResponse\nValidateShardRequest\nValidateShardResponse\nValidateKeyspaceRequest\nValidateKeyspaceResponse" | sort
ValidateKeyspaceRequest
ValidateKeyspaceResponse
ValidateRequest
ValidateResponse
ValidateSchemaKeyspaceRequest
ValidateSchemaKeyspaceResponse
ValidateSchemaShardRequest
ValidateSchemaShardResponse
ValidateShardRequest
ValidateShardResponse
ValidateVSchemaRequest
ValidateVSchemaResponse
```

Changing them to be in ☝🏿 order

Reply:
(and just to be explicit) also the rpc listing in vtctlservice.proto, and the method listing in server.go, and the test ordering in server_test.go